ee/clickhouse/queries/funnels/funnel_correlation.py

```diff
@@ -159,10 +160,33 @@ def __init__(
             # NOTE: we don't need these as we have all the information we need to
             # deduce if the person was successful or not
             include_preceding_timestamp=False,
-            include_person_properties=self.query_person_properties,
-            include_group_properties=self.include_funnel_group_properties,
+            include_properties=self.properties_to_include,
         )
 
+    @property
+    def properties_to_include(self) -> List[str]:
+        props_to_include = []
+        if self._team.actor_on_events_querying_enabled:
+            mat_event_cols = get_materialized_columns("events")
+
+            for property_name in cast(list, self._filter.correlation_property_names):
+                if self._filter.aggregation_group_type_index is not None:
+                    possible_mat_col = mat_event_cols.get((property_name, "group_properties"))
```

Comment:
why are we doing this? 🤔 . What are `group_properties` here?

Reply:
Sorry this is messed up it should be group{self._filter.aggregation_group_type_index}_properties